Library/Homebrew/dev-cmd/pull.rb

```diff
@@ -137,27 +137,33 @@ def pull
         opoo "Current branch is #{branch}: do you need to pull inside master?"
       end
 
-      patch_puller = PatchPuller.new(url, args)
-      patch_puller.fetch_patch
-      patch_changes = files_changed_in_patch(patch_puller.patchpath, tap)
-
-      is_bumpable = patch_changes[:formulae].length == 1 && patch_changes[:others].empty?
-      if do_bump
-        odie "No changed formulae found to bump" if patch_changes[:formulae].empty?
-        if patch_changes[:formulae].length > 1
-          odie "Can only bump one changed formula; bumped #{patch_changes[:formulae]}"
+      merge_commit = merge_commit?(url)
+
+      unless merge_commit
+        patch_puller = PatchPuller.new(url, args)
+        patch_puller.fetch_patch
+        patch_changes = files_changed_in_patch(patch_puller.patchpath, tap)
+
+        is_bumpable = patch_changes[:formulae].length == 1 && patch_changes[:others].empty?
+        if do_bump
+          odie "No changed formulae found to bump" if patch_changes[:formulae].empty?
```

Comment:
Make this and the two conditions below all `if`/`elsif`/else` as they all return anyway.

Reply:
Same comment as above. Can I do that in a separate PR?